src/components/UserAccount/UserAccountLastLogin.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+
+import Link from 'next/link';
+import { formatDate } from '@/helpers/dates';
+
+export function UserAccountLastLogin({
+  hrefInvite = process.env.NEXT_PUBLIC_USER_INVITE_URL || '/',
+  lastLogonTime,
+}: {
+  hrefInvite?: string;
+  lastLogonTime: number | null | undefined;
```

Comment:
[question] This doesn't quite match the expectation I have from the controller-types, where lastLongonTime is either a number or null, but it does match the controller where it can be undefined.  Should the controller-type just have an additional type of `null` on it?

Reply:
I'll bring in the `UserOrgPage` controller type to use here, so that the types are the same.